<launch>

    <arg name="model" default="$(find moveo_ps3)/urdf/moveo_ps3_end_effector.urdf"/>
    <arg name="gui" default="true" />
    <arg name="rvizconfig" default="$(find moveo_ps3)/rviz/moveo_ikpy.rviz" />

    <param name="robot_description" command="cat $(find moveo_ps3)/urdf/moveo_ps3_end_effector.urdf" />
    <param name="use_gui" value="$(arg gui)"/>

    <node name="joy" pkg="joy" type="joy_node"/>
    <node name="PS3Handler" pkg="moveo_ps3" type="PS3Handler.py" output="screen" />
    <node name="MoveoIKPy" pkg="moveo_ps3" type="MoveoIKPy.py" output="screen">
        <param name="model" value="$(arg model)" />
    </node> 
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
